account-lookup-service/Chart.yaml

```diff
@@ -12,15 +12,15 @@ maintainers:
     email: steven.oderayi@infitx.com
 dependencies:
   - name: account-lookup-service
-    version: 14.2.0
+    version: ">= 14.2.0"
```

Comment:
What is the reason for using >= for these dependencies?

Reply:
@vijayg10 I explained this change in the sprint review, but I'll recap.

We needed to use this version match format to prevent the chart update process from breaking.

Also, the version number itself is bumped when necessary as pointed out by James.